### ISSUE TITLE
[Request] want to change the comment style from "#;" which is inserted by "Ctrl + Slash"

### ISSUE THREAD

**Issue by isc-bsaviano** [intersystems-community/vscode-objectscript#1350]
### Discussed in https://github.com/intersystems-community/vscode-objectscript/discussions/1349

<div type='discussions-op-text'>

<sup>Originally posted by **seisuke24am** April 18, 2024</sup>
Both vscode-objectscript and Studio, when "Ctrl + Slash" is put in the editor window, the comment **#;** is inserted in the code.

IRIS Studio has the option that users can change this comment style from **#;** to anything as they like. 
![image](https://github.com/intersystems-community/vscode-objectscript/assets/85159985/cf1ff661-7f5e-4647-86ab-756af39db326)

Currently vscode-objectscript extention does not have this option so they cannot change the comment delimiter style. 
So I put this request to add new option to change the comment delimiters.</div>

**Comment by isc-bsaviano** [comment]
@seisuke24am In the future, please post bug reports/feature requests as issues. I don't think this is possible. VS Code's language configuration allow for one comment token per language, per type (block or line). 

@gjsjohnmurray Do you know if a way we could support this?

**Comment by gjsjohnmurray** [comment]
@isc-bsaviano I think we set it here

https://github.com/intersystems-community/vscode-objectscript/blob/e4ddce90bfc2d5b4126cd97e735872bf2f0c88ef/src/languageConfiguration.ts#L12

So it ought to be possible to support a config setting that overrides what we return to VS Code.

**Comment by isc-bsaviano** [comment]
That's true, but the language configuration is only set once at activation.

**Comment by gjsjohnmurray** [comment]
So it could be a setting that we state requires a reload.

**Comment by gjsjohnmurray** [comment]
Or try a listener for the setting change, then dispose the previous setLanguageConfiguration object and set a new one?

**Comment by isc-bsaviano** [comment]
The second approach is the one I would use. I don't feel strongly about this feature, but I'll leave it open for people to comment.

**Comment by isc-rsingh** [comment]
@seisuke24am this is an interesting idea. I would like to see additional interest before committing to find the right way to implement this. I advise requesting upvotes with a post on the [InterSystems developer community](https://community.intersystems.com/) to gain support for your feature.

**Comment by seisuke24am** [comment]
Thank you @isc-bsaviano to move my post here. I got it, I'll put similar enhancements requests here in "Issues".

Great idea @isc-rsingh , I posted the article to developer community about this request.
[https://community.intersystems.com/post/agree-or-disagree-change-comment-delimiters-vs-code-extension](https://community.intersystems.com/post/agree-or-disagree-change-comment-delimiters-vs-code-extension)
